test-runner/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Yes i think there is no need for that unless we are going to have our project merged to Apache Spark in the future.   I dont see other projects using it eg:
https://github.com/amplab/tachyon/blob/master/common/src/test/resources/log4j.properties

Apache projects do use it (its parth of the file header anyway):

eg. https://github.com/apache/curator/blob/master/curator-recipes/src/test/resources/log4j.properties
https://github.com/apache/camel/blob/master/camel-core/src/test/resources/log4j.properties